src/main/java/web/spring_boot/springbootproject/service/impl/ProductReviewServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package web.spring_boot.springbootproject.service.impl;
```

Comment:
```suggestion
package web.springboot.springbootproject.service.impl;
```
В пекедж неймінгу не використовуються underscores 
https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names